app/views/projects/index.html.erb

```diff
@@ -1,3 +1,5 @@
+<%= will_paginate %>
```

Comment:
As mentioned, I would only include this in one place (preferably at the bottom of the page).